apps/coordinator/package.json

```diff
@@ -96,6 +96,10 @@
       "<rootDir>/tests"
     ],
     "testRegex": ".*\\.test\\.ts$",
+    "moduleNameMapper": {
+      "^@excubiae/contracts$": "<rootDir>/ts/jest/__mocks__/@excubiae/contracts.ts",
+      "^@openzeppelin/merkle-tree$": "<rootDir>/ts/jest/__mocks__/@openzeppelin/merkle-tree.ts"
+    },
```

Comment:
This can be removed with added mock files, all the tests are passed without it. Renaming `ESupportedChains` should not affect any mocks or tests.

Reply:
I removed this changes and the tests broke again. Here is the cause: https://github.com/privacy-scaling-explorations/maci/pull/2653#issuecomment-3154041712

I think we should keep them

Reply:
I just rebased from main, deleted the "moduleNameMapper" and the mockfiles in `ts/jest/` and got the error:

```
npx jest ts/subgraph/__tests__/subgraph.service.test.ts --forceExit
 FAIL  ts/subgraph/__tests__/subgraph.service.test.ts
  ● Test suite failed to run

    TypeError: Cannot read properties of undefined (reading 'custom')

    > 1 | export {
        | ^
      2 |   getProxyContract,
      3 |   deployProxyClone,
      4 |   type IDeployCloneArgs,
```

<img width="1304" height="655" alt="image" src="https://github.com/user-attachments/assets/f7096f8a-97fe-4459-9693-ee18c7f8fd92" />

Reply:
I am still getting the error in a clean setup. I will remove these options and see if Github workflows passes

Reply:
Very strange but fine, let's bring it back and it's good to be merged!
Sorry for waiting!